Add dexamethasone to therapeutics (#651)
greenelab/covid19-review: content/20.treatments.md

Dexamethasone works as an anti-inflammatory by binding to glucocorticoid receptors with higher affinity than endogenous cortisol [@doi:10.1016/B978-0-323-48110-6.00034-X].
In order to understand how dexamethasone works to reduce inflammation, it is necessary to consider the stress response more broadly.
In response to stress, corticotropin‐releasing hormone stimulates the release of neurotransmitters known as catecholamines, such as epinephrine, and steroid hormones known as glucocorticoids, such as cortisol [@doi:10.1016/j.cyto.2015.01.008; @doi:10.1111/j.1749-6632.2002.tb04229.x].
While catecholamines are often associated with the characteristic fight-or-flight response, the specific role that glucocorticoids play is less clear, although they are thought to be important to restoring homeostasis [@doi:10.1159/000054578].
Immune challenge is a stressor that is known to interact closely with the stress response.

> Due to communication between the immune system and the central nervous system, macrophages can both respond to and produce catecholamines [@doi:10.1016/j.cyto.2015.01.008].

Reviewer: Would you say that this is due to communication or would you say that this is a mechanism of communication? I am a bit thrown off by the wording here. A skim of that article makes me think this is a summary of a mechanism of how they can be linked.

Author: I think it's basically that the two systems are interwoven (which makes sense since it's all stress). So maybe:
```suggestion
The immune system can therefore interact with the central nervous system; for example, macrophages can both respond to and produce catecholamines [@doi:10.1016/j.cyto.2015.01.008].
```

Additionally, the production of both catecholamines and glucocorticoids is associated with inhibition of proinflammatory cytokines such as IL-6, IL-12, and TNF‐α and the stimulation of anti-inflammatory cytokines such as IL-10, meaning that the stress response can regulate inflammatory immune activity [@doi:10.1111/j.1749-6632.2002.tb04229.x].
Administration of dexamethasone has been found to correspond to dose-dependent inhibition of IL-12 production, but not to affect IL-10 [@PMID:8902882]; the fact that this relationship could be disrupted by administration of a glucocorticoid-receptor antagonist suggests that it is regulated by the receptor itself [@PMID:8902882]. <!-- Note: waiting for full text of this paper, this is based on abstract and citations-->
Thus, the administration of dexamethasone for COVID-19 is likely to simulate the release of glucocorticoids endogenously during stress, resulting in binding of the synthetic steroid to the glucocorticoid receptor and the associated inhibition of the production of proinflammatory cytokines.

> Thus, dexamethasone reduces inflammation by stimulating the mechanism used to reduce inflammation following a threat such as immune challenge.

Reviewer: ```suggestion
In this model, dexamethasone reduces inflammation by stimulating the mechanism used to reduce inflammation following a threat such as immune challenge.
```
Maybe? trying to avoid the double thus.

> The application of dexamethasone to the treatment of COVID-19 was evaluated as part of the multi-site RECOVERY trial in the United Kingdom [@doi:10.1101/2020.06.22.20137273].

Reviewer 2: ```suggestion
The application of dexamethasone for the treatment of COVID-19 was evaluated as part of the multi-site RECOVERY trial in the United Kingdom [@doi:10.1101/2020.06.22.20137273].
```

> Over 6000 hospitalized COVID-19 patients were assigned into standard care or treatment (dexamethasone) arms of the trial with a 2:1 ratio.

Reviewer 2: ```suggestion
Over 6,000 hospitalized COVID-19 patients were assigned into standard care or treatment (dexamethasone) arms of the trial with a 2:1 ratio.
```
